src/load-script-once.js

```diff
@@ -0,0 +1,23 @@
+import loadScript from 'load-script';
+
+const scriptCache = {};
+
+/**
+ * Asynchronously load a script (once only).
+ *
+ * @param {string} options.src script source url to load.
+ * @param {string} options.integrity integrity check (sha256, sha384, sha512 hash).
+ */
+async function loadScriptOnce (options) {
+  const { src, integrity } = options;
+  if (!scriptCache[src]) {
+    scriptCache[src] = new Promise((resolve, reject) => {
+      const callback = err => err ? reject(err) : resolve();
+      loadScript(src, { attrs: { ...(integrity && { integrity }), crossorigin: 'anonymous' } }, callback);
+    });
+  }
+
+  return scriptCache[src];
+}
+
+export default loadScriptOnce;
```

Comment:
I would have thought someone had already done this exact thing on npm?

Reply:
In fact, you've got this on package.json? https://www.npmjs.com/package/load-script-once

Reply:
@timlesallen I know and I did try using that but it does not have the ability to pass options through... so if I was to use that we could not set the cors anonymous or integrity hash attributes.

Reply:
@aeh Ah!  Maybe publish yours then? :P